Add PCS video and clean up some areas (#95)
checkly/docs: constructs/private-location.mdx

---
title: 'PrivateLocation Construct'
description: 'Learn how to configure private locations with the Checkly CLI.'
sidebarTitle: 'Private Locations'
---

<Tip>
Learn more about Private Locations in [the Private Locations overview](/platform/private-locations/overview).
</Tip>

> Use Private Location to create private locations for running checks from your own infrastructure, VPCs, or segregated networks. This enables monitoring of internal services and applications not accessible from the public internet.

Reviewer: ```suggestion
Use Private Locations to run checks from your own infrastructure, VPCs, or isolated networks. With a Private Location, you can monitor internal services and applications not accessible from the public internet.
```
